cmd/utils.go

```diff
@@ -248,3 +254,78 @@ func Parse(configPath string) error {
 	}
 	return nil
 }
+
+type Argon2Params struct {
```

Comment:
shouldn't we put all these argon-specific utils into separate file? like `utils-argon2.go`?